src/popup/components/AdvancedTransactionDetails.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <DetailsItem
+    expandable
+    class="advanced-transaction-details"
+    :label="$t('transaction.advancedDetails')"
+  >
```

Comment:
I don't think this component should have this code as it forces it to have the `extra` slot. This makes the component the opposite to single-purpose.